File: plugin/src/main/java/com/violeth/blockjs/blockjs/commands/HelpCommand.java
package com.violeth.blockjs.blockjs.commands;

import com.violeth.blockjs.blockjs.BlockJS;
import com.violeth.blockjs.blockjs.Commands;
import net.kyori.adventure.text.Component;
import net.kyori.adventure.text.format.TextColor;
import org.bukkit.command.CommandSender;

public class HelpCommand extends Command {
    public HelpCommand() {
        super("help");
    }
    @Override
    public void handle(CommandSender sender, String[] args) {
        var message = Component.text("Available commands: ");

        for(var i = 0; i < Commands.list.size(); ++i) {
            var command = Commands.list.get(i);

            if(i == Commands.list.size() - 1) {
                message = message.append(Component.text(command.key).color(TextColor.color(0xFFFFFF)));
            } else {
                message = message.append(Component.text(command.key + ", ").color(TextColor.color(0xFFFFFF)));
            }
        }

        sender.sendMessage(BlockJS.getPluginMessagePrefix().append(
            message
        ));
    }
}

File: plugin/src/main/kotlin/io/github/blockjs/plugin/commands/HelpCommand.kt
package io.github.blockjs.plugin.commands

import io.github.blockjs.plugin.Main
import io.github.blockjs.plugin.Commands
import net.kyori.adventure.text.Component
import net.kyori.adventure.text.format.TextColor
import org.bukkit.command.CommandSender

class HelpCommand : Command("help") {
    override fun handle(sender: CommandSender, args: Array<out String>) {
        var message = Component.text("Available commands: ")

        for (i in 0..< Commands.list.size) {
            val command = Commands.list[i]

            if(i == Commands.list.size - 1) {
                message = message.append(Component.text(command.key).color(TextColor.color(0xFFFFFF)))
            } else {
                message = message.append(Component.text(command.key + ", ").color(TextColor.color(0xFFFFFF)))
            }
        }

        sender.sendMessage(
            Main.pluginMessagePrefix!!.append(
                message
            )
        )
    }
}
